e2e/tests/nightly/admin/user-admin-ui.spec.ts

```diff
@@ -39,90 +38,9 @@ describe('User Admin', () => {
-
-    //switch statement to validate that the status reflects the user lockout access
-    switch (userLockout) {
-      case 'DISABLE':
-        expect(status).toEqual('Active');
-        break;
-      case 'ENABLE':
-        expect(status).toEqual('Disabled');
-        break;
-    }
-    //get the name column index separately since it is a frozen column
-    const nameColIndex = await adminTable.getNameColindex();
-
-    //click on the name link to navigate to the admin-user-profile page
-    const userProfileInfo = await userAdminPage.clickNameLink(1, nameColIndex);
-
-    //navigate to admin-user-profile page
-    await userProfileInfo.waitForLoad();
-
-    //get the usernameEmail to match with the username on the admin-user-profile page
-    const userNameEmail = await userAdminPage.getUserNameEmail(1, usernameColIndex);
-
-    //verify that the save button is disabled
-    expect(await userProfileInfo.getSaveButton().isCursorNotAllowed()).toBe(true);
-
-    // get the Account Access toggle status (true or false) and the text (enabled or disabled)
-    const accountAccessToggle = userProfileInfo.getAccountAccessToggle();
-    const accountAccessStatus = await userProfileInfo.getAccountAccessText();
-
-    //switch statement to validate that the Account Access text and toggle reflects the user lockout on user-admin-table.
-    switch (userLockout) {
-      case 'DISABLE':
-        expect(accountAccessStatus).toEqual('Enabled');
-        expect(await accountAccessToggle.isChecked()).toBe(true);
-        break;
-      case 'ENABLE':
-        expect(accountAccessStatus).toEqual('Disabled');
-        expect(await accountAccessToggle.isChecked()).toBe(false);
-        break;
-    }
-
-    //verify that Name, Username and InitialCreditsUsed input fields are disabled
-    expect(await userProfileInfo.getNameInput().isCursorNotAllowed()).toBe(true);
-    expect(await userProfileInfo.getUsernameInput().isCursorNotAllowed()).toBe(true);
-    expect(await userProfileInfo.getInitialCreditsUsedInput().isCursorNotAllowed()).toBe(true);
-
-    // get the username field placeholder to verify the email on user-admin-table page and admin-user-profile page matches
-    const userNamePlaceHolder = await userProfileInfo.getUserNamePlaceHolder();
-    expect(userNameEmail).toEqual(userNamePlaceHolder);
-
-    //verify the credit limit value matches the limit value in the initial credits used field
-    const initialCreditLimit = await userProfileInfo.getInitialCreditsLimitValue();
-    const initialCreditMaxValue = await userProfileInfo.getCreditLimitValue();
-    expect(initialCreditLimit).toEqual(initialCreditMaxValue);
-
-    // verify the admin is able to update the initial credit
-    await userProfileInfo.updateInitialCredits();
-    await userProfileInfo.selectInitialCredits(initialCreditLimit);
+    await userProfileAdmin.waitForLoad();
   });
 
   test('Verify that the user-audit page UI renders correctly', async () => {
```

Comment:
This test is no longer needed. `user-profile-admin.spec` has `test('Can go to User Audit page')`

Reply:
I think there's a slight difference here, and I've renamed the 2 tests acoordingly:
"Can navigate to the UserAuditPage from the UserAdminPage" 
"Can navigate to the UserAuditPage from the UserProfileAdminPage"